kubernetes/run-e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Call this (and the other) .sh file in the appropriate way during `make test`, and check that that causes no issues if a cluster is not up.